public/stylesheets/style.css

```diff
@@ -1,9 +1,93 @@
 body {
   padding: 50px;
   font: 14px "Lucida Grande", Helvetica, Arial, sans-serif;
-  
+
+}
+
+.modalPost {
+  display: flex;
+  flex-direction: column;
+}
+
+.buttonControls {
+  display: flex;
+  flex-direction: row;
+  align-items:flex-end;
 }
 
 a {
-  color: #00B7FF;
+  text-decoration: none;
+}
+
+.fa.fa-trash-o {
+  cursor: pointer;
+}
+.buttonControls .fa {
+  margin-right: 1%;
+}
+.imageContainer img {
+  width: 300px;
+  height: 200px;
 }
+.createInput {
+  margin: 3% 0;
+}
+
+.formFields {
+  display: flex;
+  flex-direction: column;
+}
+
+.modalDialog {
+        position: fixed;
+        font-family: Arial, Helvetica, sans-serif;
+        top: 0;
+        right: 0;
+        bottom: 0;
+        left: 0;
+        background: rgba(0,0,0,0.8);
+        z-index: 99999;
+        opacity:0;
+        -webkit-transition: opacity 400ms ease-in;
```

Comment:
I recommend investigating CSS pre-processors (like [Autoprefixer](https://github.com/postcss/autoprefixer)) to avoid having to explicitly define browser specific prefixes.  It requires the addition of tooling to your project, but saves a ton of time and effort figuring out how to get your CSS to apply across browsers.